web/src/pages/Home/useMenuDeleteCallback.tsx

```diff
@@ -0,0 +1,15 @@
+import {useCallback} from 'react';
+import {MenuInfo} from 'rc-menu/lib/interface';
+import {ITest} from '../../types/Test.types';
+
+export function useMenuDeleteCallback(
```

Comment:
I don't think we need a new file, just having this function above the return section should be enough 😄

Reply:
All the projects that I have been working lately follow this standard. 
Having a single components/function per file should be something to aim for.
Abstracting hooks to different files also helps with readability of a component render method. 
Also help with its file size. React components should not be bigger than 50 lines.
Should I merge this? whats the correct procedure here? @xoscar

Reply:
@xoscar I think we should chat and come up with a Frontend standard to follow from now on

Reply:
I think components should have the basic functionality being part of it, then it would be harder to follow up on what is going on. I agree with you that we have to keep components small tho. That's why most of the business logic is encapsulated in the services and models.